calico_containers/calico_ctl/node.py

```diff
@@ -67,7 +67,8 @@
 
 POLICY_ONLY_ENV = "POLICY_ONLY_CALICO"
 
-KUBERNETES_BINARY_URL = 'https://github.com/projectcalico/calico-docker/releases/download/v0.5.4/calico_kubernetes'
+PLUGIN_VERSION = 'v0.1.0'
```

Comment:
Name this KUBERNETES_PLUGIN_VERSION  or KUBE_PLUGIN_VERSION -- There may be other plugins.